aten/src/ATen/native/MathBitsFallback.h

```diff
@@ -44,6 +44,9 @@ struct MathOpFallback {
     const auto stack_start = stack->size() - num_arguments;
 
     c10::optional<bool> is_write;
+    // Mutable inputs to be tracked separately
+    std::vector<Tensor> mutable_inputs;
+    bool check_for_alias_with_mut_arg = false;
```

Comment:
A good comment here would be an explanation of all the things we precompute. Previously, we just tested if the function overall was a "write" function (though arguably should have explained what `is_write` meaning was); now there's three things we are precomputing and so it is a good idea to just generally say what the semantics of these variables are.

Reply:
In particular, `mutable_inputs` here only include *conjugated* mutable inputs. Variable name here doesn't make this clear at all!

Reply:
Similarly, `check_for_alias_with_mut_arg` doesn't seem necessary at all; it is always true when `mutable_inputs` is non-empty